README.md

```diff
@@ -1 +1,15 @@
 # Enceladus's Github Pages branch
+
+This branche's sole purpose is to serve as Documentation for the Enceladus project
+
+### Dependencies:
+- ruby >= 2.3.0
+
+### To build locally:
+```bash
+# In the root of the project
+$> git checkout gh-pages
+$> bundle install
+$> bundle exec jekyll serve
+# => Now browse to http://localhost:4000
```

Comment:
Make this a actual clickable link in the `readme.md`

Reply:
Again, I would rather not. This would point to a "site" that is available only at a certain point after you do everything needed. It is not a site that is always available or something. It can even lead people to wrong sites that they have previously built and left running.

PS: These 3 lines were "stolen" from official `Jekyll` README. They also don't use link